packages/icon-replacer-tool/icon_mappings.json

```diff
@@ -276,5 +276,6 @@
   "person_grey_child.png": "avatar__child--125-enabled.svg",
   "icon_days_left.png": "days-left—20-enabled.svg",
   "icon_reaching_time_limit.png": "days-reaching-limit--20-enabled.svg",
-  "icon_reaching_time_limit_hover.png": "days-reaching-limit--20-enabled.svg"
+  "icon_reaching_time_limit_hover.png": "days-reaching-limit--20-enabled.svg",
+  "days-left—20-enabled.svg": "days-left-20-enabled.svg"
```

Comment:
The key/filename on the left should be the old icon filename that is being replaced.

If you are not _replacing_ an existing icons, then you don't need to add an entry into this mapping.

Also to note, this icon is already in the mappings on line 277 but the filename is incorrect. It's missing a double dash between "left" and "20".